e-keithel-backend/config/routes.rb

```diff
@@ -1,6 +1,3 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resource :users, only: [:create]
```

Comment:
Maybe there should be option for deleting users down the line. So, we should have destroy action too. But it can wait!